tweets/conference.tweet

```diff
@@ -0,0 +1,9 @@
+🏗 Building Network States w/ @buildcities 
+
+We are proud to announce that Zenon is the head-sponsor of this event.
+
+🚶 Attend in person <1 mile from @EthereumDenver, or watch the recording on-demand 📺
```

Comment:
Should this be?

Attend in person, it's less than 1 mile from @EthereumDenver,